src/tap/README.md

```diff
@@ -24,6 +24,47 @@ tapcell [-tapcell_master tapcell_master]
         [-endcap_prefix endcap_prefix]
 ```
 
+- `-tapcell_master`. Specify the master used as a tapcell.
+- `-endcap_master`. Specify the master used as an endcap.
+- `-distance`. Specify the distance (in microns) between each tapcell in the checkerboard.
+- `-halo_width_x`. Specify the horizontal halo size (in microns) around macros during cut rows.
+- `-halo_width_y`. Specify the vertical halo size (in microns) around macros during cut rows.
+- `-tap_nwintie_master`. Specify the master cell placed at the top and bottom of
+macros and of the core area, according the row orientation.
```

Comment:
Remove the second "of" and the comma:
Specify the master cell placed at the top and bottom of macros and ~of~ the core area according to the row orientation.

Review all occurrences in the document.